index.html

```diff
@@ -0,0 +1,1975 @@
+                <li><a href="#">GROWTH QUARTERS</a></li>
+              </ul>
+            </div>
+            <img class="sm-logo lg-hid m-hid" src="./assets/Images/small-nav-logo.jpg" alt="logo">
+            <div class="right-nav as-center">
+              <ul class="d-flex">
+                <li><a href="#"><i class="fas fa-search"></i></a></li>
+                <li><a href="#"><i class="fas fa-bars lg-hid"></i></a></li>
+                <li><a class="sm-hid m-hid" href="#"><img src="./assets/Images/white-nav-logo.jpg" alt="logo"
+                      class="white-nav-logo"></a></li>
+                <li><a href="#" class="countdown sm-hid m-hid">21 : 20 :25 :24</a></li>
+                <li><a href="#" class="tickets-btn sm-hid m-hid">GET TICKETS</a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="yellow-navbar w-100 d-flex">
+        <a href="#" class="as-center black">Join TNW2020, our 15th annual flagship event. Online. Get tickets →</a>
+      </div>
+    </div>
+  </nav>
+  <!-- NEW ARTICLE -->
+  <header>
+    <div class="article-1-grid">
+      <div class="div-grid-1 cp d-flex">
+
+        <a href="#"><img src="assets/Images/TNW-3.svg" alt="" class="TNW"></a>
+        <div class="div-grid-1-date">
+          <p>Monday - Septemeber 7, 2020</p>
+        </div>
+
+        <div class="div-little-text">
+          <p class="main0title">Scientists demonstrate first 'plug and play' brain prosthesis in a paralyzed person</p>
+          <p class="litlle-p">by <a class="cp" href="#">Thomas Macaulay</a>&nbsp;&nbsp;<i
+              class="far fa-clock"></i>&nbsp;
+            4 hours ago &nbsp;<i class="fas fa-share-alt">&nbsp;</i>&nbsp;7</p>
+        </div>
+      </div>
+      <div class="div-grid-2 cp d-flex">
+        <div class="div-grid-2-p">
+          <p>Join our 15th annual flagship event for free</p>
+        </div>
+      </div>
+      <div class="div-grid-3 cp d-flex">
+        <div class="div-grid-3-p">
+          &nbsp;<p class="low">DISTRACT</p>
+        </div>
+        <div class="grid-3-p">
+          <p class="div-grid-3-container">Watch Ken Block tear <br> it up in an all-electric Rallycross car</p>
+        </div>
+        <div class="div-grid-3-artist">
+
+          <p>by <a href="#">Matthew Beedham</a>&nbsp;<i class="far fa-clock"></i>&nbsp;6 hours ago&nbsp; <i
+              class="fas fa-share-alt"></i>&nbsp;7</p>
+        </div>
+      </div>
+
+    </div>
+  </header>
+  <!-- NEW SECTION -->
+  <main>
+    <article class="section-2">
+      <h4 class="l-h4 cp">Latest news &nbsp;<i class="fas fa-chevron-right"></i></h4>
+      <div class="grid-article-2">
+
+        <div class="div-g-1 cp">
+
+          <img src="assets/Images/kreten-1.jpg" alt="" class="slike">
+          <p class="main">Tesla dumped $5B worth of its stock in just 4 days — and traders hate it</p>
+          <p class="main-little">David Canellis4 hours ago</p>
```

Comment:
- [ ] The height in the tablet screen size should be a little smaller.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/93134880-c8651980-f6af-11ea-9e41-55349553a2fc.png)| ![image](https://user-images.githubusercontent.com/61479648/93134920-db77e980-f6af-11ea-936f-1119a50bcdec.png) |